tests/test-nim-to-wasm.ts

```diff
@@ -34,8 +34,9 @@ describe('calling Nim functions compiled to Wasm', () => {
   beforeAll(async () => {
     await Promise.all(
       filesToTest.map(async nimFilePath => {
-        const wasmFilePath = (await compileNimFileToWasm(nimFilePath))
-          .outputFileName;
+        const wasmFilePath = (
+          await compileNimFileToWasm(nimFilePath, '--d:lightClientWASM')
+        ).outputFileName;
         const exports = await loadWasm<{}>({
           from: {
             filepath: wasmFilePath,
```

Comment:
The Nim change below doesn't seem suitable for upstreaming. I think we'll need to propose a way for the user to be able to specify a custom module providing `malloc` definitions trough a Nim compile-time options such as `-d:userMallocModule=my-malloc.nim`